_includes/tag_list.html

```diff
@@ -1,4 +1,10 @@
 <div id="sidebar">
+  <div id="advertisement">
+    <a href="https://fastruby.io" target="_blank">
+      <img id="fastruby" src="assets/images/upgrade_rails.png" alt="Upgrade Ruby">
```

Comment:
@ercohen14 I think the alt should say "Upgrade your Ruby on Rails app with fastruby.io"